common/src/common/messages.py

```diff
@@ -298,6 +327,12 @@ async def cans_send(
     await socket.send(serial)
 
 
+def get_user_message_cookie(receiver: str, text: str) -> str:
```

Comment:
why do we call it cookie? Makes no sense to me

Reply:
Why not token, id, fingerprint etc? Isn't cookie a piece of data that we store clientside and later send to the server in order to have some kind of session-like behaviour in a stateless protocol such as HTTP? Maybe we do that here, Im not sure
Also, is this really unique? Is it just receiver and text? Shouldn't we add timestamp or something? It's not uncommon to have messages like "ok", "no", "yeah" etc. Is it taken into consideration in the `text` param?